content/tech-big-picture/dev-ops/_index.md

```diff
@@ -19,5 +19,5 @@ By fostering a culture of collaboration and shared responsibility, DevOps practi
 
 ## Resources 
 
-- https://www.nimblework.com/blog/introduction-to-devops/
-- https://www.geeksforgeeks.org/introduction-to-devops/
\ No newline at end of file
+- [Introduction to DevOps](https://www.nimblework.com/blog/introduction-to-devops/)
```

Comment:
Should these titles be the same? If the learners see 2 links with the same title then they will probably only click on one of them or assume we made an error.

Maybe say
- Nimblework introduction to devops
- geeks for geeks introduction to devops